internal/pkg/cli/project_delete.go

```diff
@@ -182,26 +212,58 @@ func (o *deleteProjOpts) deleteEnvs() error {
 	return nil
 }
 
-func (o *deleteProjOpts) deleteApps() error {
-	apps, err := o.store.ListApplications(o.ProjectName())
+func (o *deleteProjOpts) emptyS3Bucket() error {
+	proj, err := o.store.GetProject(o.ProjectName())
+	if err != nil {
+		return fmt.Errorf("get project %s: %w", o.ProjectName(), err)
+	}
+	projResources, err := o.projResourcesGetter.GetRegionalProjectResources(proj)
 	if err != nil {
+		return fmt.Errorf("get regional resources for %s: %w", proj.Name, err)
+	}
+	for _, projResource := range projResources {
+		o.spinner.Start(fmt.Sprintf("Deleting S3 bucket objects in %s.", projResource.Region))
```

Comment:
can we add the name of the bucket? maybe something like
```suggestion
		o.spinner.Start(fmt.Sprintf("Emptying the pipeline bucket %s in %s.", color.HighlightResource(projResource.S3Bucket), projResource.Region))
```

Reply:
Nice!